components/nifosg/nifloader.cpp

```diff
@@ -1340,13 +1359,15 @@ namespace NifOsg
                 }
                 break;
             }
-            // unused by mw
-            case Nif::RC_NiShadeProperty:
-            case Nif::RC_NiDitherProperty:
-            case Nif::RC_NiFogProperty:
-            {
+            // These three are unused by mw, and reporting them causes log spam from the original assets
+            case Nif::RC_NiShadeProperty:   //Flag to enable smooth phong shading (we already do this)
+            case Nif::RC_NiDitherProperty:  //Flag to enable dithering
                 break;
-            }
+            case Nif::RC_NiFogProperty:     //Add a fog emitter to the object
```

Comment:
As per the last comment, not really a fog emitter.

I must say that I don't see the point in commenting unused properties. Since the properties were never utilized, it is impossible to say what they were really supposed to do. Best to just ignore them and move on. So remove the comments please.